contributing/development.md

```diff
@@ -54,7 +54,7 @@ Today it's a dynamic Node.js webserver powered by Express, using [middleware](..
 
 The tooling for this site has changed over the years, but many of the tried-and-true authoring conventions of the original Jekyll site have been preserved:
 
-- Content is written in Markdown files, which live in the `content` directory.
+- Content is written in Markdown files, which are live in the `content` directory.
```

Comment:
```suggestion
- Content is written in Markdown files, which live in the `content` directory.
```
The current wording here is correct, so I'm going to remove the "are" from this line.